tools/sof-get-default-tplg.sh

```diff
@@ -3,9 +3,9 @@
 # Current: load it from dmesg
```

Comment:
could you change the comment here?
I see it will fallback in below order:
```
dmesg
journalctl -k
/var/log/syslog
```

Reply:
Marking resolved as @Bin-QA made the change with the latest push.